docs/src/main/sphinx/connector/query-function-escaping-quotes.fragment

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+  If your query statement includes characters in quotes, you must escape the
```

Comment:
This is just how strings work in SQL. We may want to add documentation about that in the language reference section and just link to it.

Reply:
If it's not anything Trino specific then my vote's on not bothering with it in our docs. I thought initially that this was something that would specifically trip up folks using the `query` function, but if not then I think we can just close this PR. @kasiafi do you have any thoughts on this?

Reply:
I think we should add it since Trino users are often SQL newbies .. but it should be in the SQL language docs..